10/RequestReply.md

```diff
@@ -103,15 +103,40 @@ NOTE All GROUPED REQUESTS should be logged once with the same information as the
 
 ### Connection management
 
-* The Reqest-Reply protocol SHALL manage client connection.
+The Request-Reply protocol SHALL manage client connections to Device Servers.
 
 #### Connection establishement
 
+* To send any REQUESTS to a Device, a Client MUST connect to the Device Server manging the Device.  
+* A Device Server SHALL wait for client connections on one or multiple network endpoints.
+* The network endpoint SHALL define \<network_address\> and \<tcp_port\>, where:
+
+... abnf
+ network_address = ip_address / host_name
+ tcp_port = num-val
+...
+
+* A Client MAY connect any Device Server on one of Device Server's endpoints.
+* If a Client is running on the same host as a Device Server, the client MAY use other than
```

Comment:
This is specific to the CORBA protocol so I would say the way to connect is delegated to the underline protocol. It means that this RFC should say explicitly what is an underline protocol.

Reply:
So, I propose to remove this point.